templates/download/cloud/autopilot.html

```diff
@@ -1,269 +1,249 @@
-        <div class="five-col">
-          <h3 class="list-stepped__title">Review your checklist</h3>
-        </div>
-        <div class="seven-col last-col list-stepped__box right">
-          <p>At the bottom of the setup page there is a checklist with the status of all of your resources. These should all be green, if it isn’t follow the instructions to resolve.</p>
-        </div>
-      </li>
-      <li class="list-stepped__item">
-        <div class="five-col">
-          <h3 class="list-stepped__title">Choose your OpenStack components</h3>
-        </div>
-        <div class="seven-col last-col list-stepped__box right">
-          <img src="{{ ASSET_SERVER_URL }}32a02c50-install-openstack-with-autopilot-step8.png" alt="" />
-        </div>
-      </li>
-      <li class="list-stepped__item">
-        <div class="five-col">
-          <h3 class="list-stepped__title">Select the hardware on which to deploy the cloud</h3>
-        </div>
-        <div class="seven-col last-col list-stepped__box right">
-          <img src="{{ ASSET_SERVER_URL }}30c17ac2-install-openstack-with-autopilot-step9.png" alt="" />
-        </div>
-      </li>
-      <li class="list-stepped__item">
-        <div class="five-col">
-          <h3 class="list-stepped__title">Select &ldquo;Install&rdquo; to start building your cloud</h3>
-        </div>
-        <div class="seven-col last-col list-stepped__box right">
-          <img src="{{ ASSET_SERVER_URL }}77bc020b-Installing+region+1++++table+++Progress.png" alt="" />
-        </div>
-      </li>
-      <li class="list-stepped__item">
-        <div class="five-col">
-          <h3 class="list-stepped__title">Create an OpenStack account to access your Horizon dashboard</h3>
-        </div>
-        <div class="seven-col last-col list-stepped__box right">
-          <img src="{{ ASSET_SERVER_URL }}19a76480-while-you-wait.png" alt="" /><br/><br/><br/>
-        </div>
-      </li>
-      <li class="list-stepped__item">
-        <div class="five-col">
-          <h3 class="list-stepped__title">Monitor your region and scale out</h3>
-        </div>
-        <div class="seven-col last-col list-stepped__box right">
-          <img src="{{ ASSET_SERVER_URL }}7405e963-install-openstack-with-autopilot-step12.png" alt="" />
-        </div>
-      </li>
-    </ol>
+      </div>
+    </div>
   </div>
-</section>
+</div>
 
-<section class="row row-white no-border">
-  <div class="strip-inner-wrapper equal-height">
-    <div class="four-col align-center not-for-small equal-height__item equal-height__align-vertically">
-      <img src="{{ ASSET_SERVER_URL }}1f1d581a-picto-quote-orange.svg" width="200" alt="" />
+{% include "download/shared/_get_ebook-v1.html"%}
+
+<div class="p-strip--light is-deep is-bordered" id="instructions">
+  <div class="row">
+    <div class="col-12">
+      <h2>Installation instructions</h2>
+
+      <ol class="p-stepped-list--detailed">
+
+        <li class="p-list-step__item col-12">
```

Comment:
`col-12` should not be used like this - and in this case (and all other `.p-list-step__item`), it seems redundant anyway?